app/views/layouts/application.html.erb

```diff
@@ -54,24 +54,25 @@
                 class="govuk-header__logotype-crown"
                 xmlns="http://www.w3.org/2000/svg"
                 viewbox="0 0 132 97"
-                height="31"
-                width="35">
+                height="30"
```

Comment:
just checking, isn't the height 32? https://github.com/alphagov/product-page-example/blob/master/source/stylesheets/modules/_govuk-logo.scss

Reply:
I aligned it with what we have in the product page repo, but happy to align onto whichever is the correct one. :triangular_ruler:

Reply:
I defer it to you, you will know best what is the correct one. I am happy either way, if this is what's on the product page also.